karma-cov.conf.js

```diff
@@ -34,16 +34,17 @@ module.exports = function (config) {
   karma_config.specReporter = {suppressPassed: true, suppressSkipped: true};
   karma_config.coverageReporter = {
     reporters: [
-      {type: 'html', dir: 'dist/coverage/', subdir: subdir_name},
+      // {type: 'html', dir: 'dist/coverage/', subdir: subdir_name},
```

Comment:
Unless there's a good reason, we should probably remove unused lines rather than commenting them out.

Reply:
Will do.  Thanks.